draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +448,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
- |      supported-mode-id                                          |
- |      organization-id (e.g., ITU-T, OIF, OpenConfig or Vendor)   |
- |      tag* // Optional. It has various usage in future           |
- |      operational-mode                                           |
- |              capability-attribute-1                             |
- |              capability-attribute-2                             |
- |              ....                                               |
- |              capability-attribute-n                             |
+ |  optical-channel  // OTSi channels                              |
+ |      configuration // list of R/W plug configuration attributes |
+ |           config-attribute-1                                    |
+ |           config-attribute-2                                    |
+ |           .....                                                 |
+ |           config-attribute-m                                    |
  |-----------------------------------------------------------------|
 
 ~~~~
-{: #figure-plug-supported-mode title="Data structure for Coherent pluggable supported-modes"}
-
-As an example, the operational-mode 0x3E introduced by OIF contains the following read-only capability attributes. Notes that in general a coherent pluggable can support multiple operational modes.
-
-~~~~
-organization-id:  OIF
-operational-mode: 0x3E
-      modulation: DP-16QAM
-      bit-rate: 478.75 Gbps
-      baud-rate: 59.84375 Gbd
-      more attributes ...
-~~~~
+{: #figure-plug-config title="Data structure for Coherent pluggable Configuration Attributes"}
 
+{: #plug-pm-definition}
+## Coherent Pluggable Performance Monitoring Data
 
-{: #plug-config-attribute}
-## Coherent Pluggable Configurations Attributes
-
-Referring to {{figure-plug-config-pm}}, the coherent pluggables support a set of read-write attributes which are configurable. Example of such configuration attributes are output power, central frequency and operational-mode. Note that as discussed in {{plug-capabilities-attributes}}, since a coherent pluggable may support multiple operational-modes, as part of these configuration attributes, operator should configure which of these operational-mode is desired and should be functional.
+{{figure-plug-pm}} shows the list of pluggable Performance Monitoring (PM) and state data, which are critical components in optical networks, enabling network engineers to ensure optimal performance, identify issues, and maintain network reliability. Operators monitor a range of attributes on both the optical/photonic and electrical sides of coherent pluggables, including channel input power, channel output power, central frequency, current Optical Signal-to-Noise Ratio (OSNR), Bit Error Rate (BER), chromatic dispersion, laser temperature, link status, and more. These parameters directly impact the quality and integrity of the transmitted data across both optical and electrical domains.
 
```

Comment:
line 481
{{figure-plug-pm}} shows the list of optical module Performance Monitoring (PM) and state data, which are critical components in optical networks, enabling network engineers to ensure optimal performance, identify issues, and maintain network reliability. Operators monitor a range of attributes on both the optical/photonic and electrical sides of coherent modules, including channel input power, channel output power, central frequency, current Optical Signal-to-Noise Ratio (OSNR), Bit Error Rate (BER), chromatic dispersion, laser temperature, link status, and more. These parameters directly impact the quality and integrity of the transmitted data across both optical and electrical domains.